### DISCUSSION TITLE
Need support about Artemis system

### DISCUSSION THREAD

**Discussion by Serena1227** [ls1intum/Artemis#4733] [Q&A] [answered]
Hello, I want to ask about Artemis system. When admin create courses 👉🏻 Then create some exercises or exams 👉🏻 After that, copy the courses link (or exercises link) and send it to users 👉🏻 But when users click the link 👉🏻 System shows nothing 👉🏻 The users can’t see anything about the courses or exercises, and they can’t register for the courses. Do we have any solutions for this?

**Comment by krusche** [comment, marked as answer]
You can manually add users as students to the course. Go to course administration, and click on XX students.
Alternatively, you can allow students to register for the course when editing the course. You can define a pattern for users which should be able to register in the yml file.

One question: in which institution do you use Artemis?
